doc/api/http2.md

```diff
@@ -30,6 +30,41 @@ can be accessed using:
 const http2 = require('http2');
 ```
 
+## Determining if crypto support is unavailable
+
+It is possible for Node.js to be built without including support for the
+`crypto` module. In such cases, attempting to `import` from `http2` or
+calling `require('http2')` will result in an error being thrown.
+
+When using CommonJS, the error thrown can be caught using try/catch:
+
+```cjs
+let http2;
+try {
+  http2 = require('http2');
+} catch (err) {
+  console.log('http2 support is disabled!');
+}
+```
+
+When using the lexical ESM `import` keyword, the error can only be
+caught if a handler for `process.on('uncaughtException')` is registered
+_before_ any attempt to load the module is made -- using, for instance,
+a preload module.
+
+When using ESM, if there is a chance that the code may be run on a build
+of Node.js where crypto support is not enabled, consider using the
+`import()` function instead of the lexical `import` keyword:
```

Comment:
FTR, other indicators of crypto being present/unavailable are
- `process.config.variables.node_use_openssl`
- `process.features.tls`

but it probably isn't a good idea to be encouraging use of those (particularly `process.features` which AFAIK remains undocumented).